src/sentry/integrations/gitlab/client.py

```diff
@@ -319,14 +323,16 @@ def check_file(self, repo: Repository, path: str, ref: str) -> str | None:
             encoded_path = quote(path, safe="")
 
             request_path = GitLabApiClientPath.file.format(project=project_id, path=encoded_path)
-            return self.head_cached(request_path, params={"ref": ref})
+            return self.head_cached(request_path, params={"ref": version})
         except ApiError as e:
             # Gitlab can return 404 or 400 if the file doesn't exist
             if e.code != 400:
```

Comment:
Should this include 404 too then because of the comment?

Reply:
ah yes, i can fix this here i guess